### PR TITLE
Add ability to write prompts for genai at the camera level

### LINKED ISSUE #13740: Request: camera specific prompts for genai
I'd like to provide some prior description of what my camera is pointing at to improve the text descriptions. 

So if I could prepend a camera specific description to the prompt I think that would help. Something like "this camera is in a garage with a door to the house on the left".

Another idea is to pass the AI model a “nominal frame” so it can focus things that are different in the scene.  This nominal frame would be fixed per camera and taken in good lighting, etc.

### PR DESCRIPTION
**Pull request by hawkeye217**
Closes https://github.com/blakeblackshear/frigate/issues/13740

### CODE REVIEW

File: docs/docs/configuration/reference.md [hunk 1]
```diff
@@ -712,6 +712,18 @@ cameras:
       # By default the cameras are sorted alphabetically.
       order: 0
 
+    # Optional: Configuration for AI generated tracked object descriptions
+    genai:
```
Comment (NickM-27) [inline docs/docs/configuration/reference.md:716]: better to not include there here and instead just put that it can be overridden at the camera level

Comment (hawkeye217) [reply docs/docs/configuration/reference.md:716]: Only thing is that the options are different at the camera level, you can't have `model`, `provider`, etc that the root level has. So I just copied what we did for birdseye, which is also in the reference at the camera level.

Comment (NickM-27) [reply docs/docs/configuration/reference.md:716]: I don't see birdseye referenced at the camera level

Comment (hawkeye217) [reply docs/docs/configuration/reference.md:716]: https://github.com/blakeblackshear/frigate/blob/0dc72173f4cfd49a044db33df4c16cc7887839ce/docs/docs/configuration/reference.md?plain=1#L710

Comment (NickM-27) [reply docs/docs/configuration/reference.md:716]: lol, the mark for control + f was hidden by an overlay. 👍

**Review by NickM-27 — APPROVED** [review]

**Pull request merged**